### PR TITLE
TilingSprite bounds fixes

### PR DESCRIPTION
**Pull request by ivanpopelyshev**
and a test

### CODE REVIEW

File: test/core/TilingSprite.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+describe('PIXI.Sprite', function ()
```
Comment (englercj) [inline test/core/TilingSprite.js]: TilingSprite

File: test/core/TilingSprite.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+{
+    describe('getBounds()', function ()
+    {
+        it('must have correct value according to _width and _height', function ()
```
Comment (englercj) [inline test/core/TilingSprite.js]: Looks like this test is actually failing:

```
1) PIXI PIXI.Sprite getBounds() must have correct value according to _width and _height:
      AssertionError: expected -350 to equal -150
      + expected - actual
      --350
      +-150
      
      at Context.<anonymous> (/home/travis/build/pixijs/pixi.js/test/core/TilingSprite.js:21:33)
      at runCallback (timers.js:574:20)
```

Comment (ivanpopelyshev) [inline test/core/TilingSprite.js]: That means we have bugs in PIXI.Sprite too :) I'm fixing it.

Comment (englercj) [inline test/core/TilingSprite.js]: The failing test is in the TilingSprite test case, not the Sprite one. It was just incorrectly named, but you can see it comes from the TilingSprite test:

```
/home/travis/build/pixijs/pixi.js/test/core/TilingSprite.js:21:33
```

Comment (ivanpopelyshev) [inline test/core/TilingSprite.js]: I fixed both. Also renamed testset.

**Pull request merged**